lib/index.js

```diff
@@ -133,6 +136,15 @@ MapboxGeocoder.prototype = {
       this._map.on('moveend', this._updateProximity);
     }
 
+    this.mapMarker = null;
+    this._handleMarker = this._handleMarker.bind(this);
+
+    this._mapboxgl = this.options.mapboxgl;
+    if (!this._mapboxgl) {
+      console.error("No mapboxgl detected. Map markers are disabled. Please set options.mapboxgl.");
```

Comment:
Should we only log a console error if this.marker is not null? After all, if we aren't setting markers, then we don't need the options.mapboxgl so it's not actually an error.